sql-parser/src/main/java/io/crate/sql/ExpressionFormatter.java

```diff
@@ -586,35 +584,35 @@ protected String visitTryCast(TryCast node, @Nullable List<Expression> parameter
 
         @Override
         protected String visitSearchedCaseExpression(SearchedCaseExpression node, @Nullable List<Expression> parameters) {
-            ImmutableList.Builder<String> parts = ImmutableList.builder();
+            ArrayList<String> parts = new ArrayList<>();
             parts.add("CASE");
             for (WhenClause whenClause : node.getWhenClauses()) {
                 parts.add(whenClause.accept(this, parameters));
             }
             if (node.getDefaultValue() != null) {
-                parts.add("ELSE")
-                    .add(node.getDefaultValue().accept(this, parameters));
+                parts.add("ELSE");
+                parts.add(node.getDefaultValue().accept(this, parameters));
             }
             parts.add("END");
 
-            return "(" + String.join(" ", parts.build()) + ")";
+            return "(" + String.join(" ", parts) + ")";
         }
 
         @Override
         protected String visitSimpleCaseExpression(SimpleCaseExpression node, @Nullable List<Expression> parameters) {
-            ImmutableList.Builder<String> parts = ImmutableList.builder();
-            parts.add("CASE")
-                .add(node.getOperand().accept(this, parameters));
+            ArrayList<String> parts = new ArrayList<>();
```

Comment:
Could also use a StringBuilder or StringJoiner directly here and further above

Reply:
i went for StringBuilder